content/zh/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md

```diff
@@ -1,83 +1,99 @@
+title: Articles on dockershim Removal and on Using CRI-compatible Runtimes
 content_type: reference
 weight: 20
 -->
 
 <!-- overview -->
 <!-- 
-This is a list of articles about:
-
-	- the Kubernetes' deprecation and removal of _dockershim_
-	- using CRI-compatible container runtimes
+This is a list of articles and other pages that are either
+about the Kubernetes' deprecation and removal of _dockershim_,
+or about using CRI-compatible container runtimes, in connection
+with that removal.
 -->
-这是有关以下内容的文章列表：
-
-- Kubernetes 弃用和删除 _dockershim_
-- 使用兼容 CRI 的容器运行时
+这是关于 Kubernetes 弃用和删除 “dockershim”
+或使用兼容 CRI 的容器运行时相关的文章和其他页面的列表，
 <!-- body -->
 
 <!-- 
-## Primary sources
+## Kubernetes project
+
+* Kubernetes blog: [Dockershim Removal FAQ](/blog/2022/02/17/dockershim-faq/) (originally published 2022/02/17)
 
-* [Kubernetes Blog: "Dockershim Deprecation FAQ", 2020/12/02](/blog/2020/12/02/dockershim-faq/)
+* Kubernetes blog: [Kubernetes is Moving on From Dockershim: Commitments and Next Steps](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/) (published 2022/01/07)
 
-* [Kubernetes Documentation: "Migrating from dockershim"](/docs/tasks/administer-cluster/migrating-from-dockershim/)
+* Kubernetes blog: [Dockershim removal is coming. Are you ready?](/blog/2021/11/12/are-you-ready-for-dockershim-removal/) (published 2021/11/12)
 
-* [Kubernetes Documentation: "Container runtimes"](/docs/setup/production-environment/container-runtimes/)
+* Kubernetes documentation: [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/)
 
-* [Kubernetes enhancement issue: "Removing dockershim from kubelet" (`kubernetes/enhancements#2221`)](https://github.com/kubernetes/enhancements/issues/2221)
+* Kubernetes documentation: [Container runtimes](/docs/setup/production-environment/container-runtimes/)
 
-* [Kubernetes enhancement proposal: "KEP-2221: Removing dockershim from kubelet"](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2221-remove-dockershim/README.md)
+* Kubernetes enhancement proposal: [KEP-2221: Removing dockershim from kubelet](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2221-remove-dockershim/README.md)
 
-* [Kubernetes Blog: "Dockershim removal is coming. Are you ready?", 2021/11/12](/blog/2021/11/12/are-you-ready-for-dockershim-removal/)
+* Kubernetes enhancement proposal issue: [Removing dockershim from kubelet](https://github.com/kubernetes/enhancements/issues/2221) (_k/enhancements#2221_)
 -->
 
-## 首要来源
+## Kubernetes 项目
+
+* Kubernetes 博客：[Dockershim 弃用常见问题解答](/blog/2022/02/17/dockershim-faq/)（最初发表于 2022/02/17）
+
+* Kubernetes 博客：[Kubernetes 即将移除 Dockershim：承诺和下一步](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/)（发表于 2022/01/07）
 
-* [Kubernetes 博客: “Dockershim 弃用常见问题解答”, 2020/12/02](/blog/2020/12/02/dockershim-faq/)
+* Kubernetes 博客：[移除 Dockershim 即将到来。你准备好了吗？](/blog/2021/11/12/are-you-ready-for-dockershim-removal/)（发表于  2021/11/12）
```

Comment:
This article has Mandarin version too.
<https://kubernetes.io/zh/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/>